companion/src/modeledit/setup.cpp

```diff
@@ -238,9 +238,9 @@ ModulePanel::ModulePanel(QWidget * parent, ModelData & model, ModuleData & modul
   lock = false;
 
   // a change will not register if only one item was added
-  if (ui->protocol->count() == 1)
+  if (ui->protocol->count() == 1) {
```

Comment:
Doesn't matter.  I just didn't want you or anyone to feel like they should be cleaning up.  I used no-brackets syntax a bunch in those changes, so if it bothers anyone just let me know.  If some instances are less readable than others and can use brackets, that's fine with me also.